packages/go/util/analyze.go

```diff
@@ -104,84 +103,6 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	se := string(rf)
-
-	var files []string
-	var relatedFiles []string
-
-	// Add entrypoint to watchlist
-	relFileName, err := filepath.Rel(filepath.Dir(fileName), fileName)
-	if err != nil {
-		log.Fatal(err)
-	}
-	relatedFiles = append(relatedFiles, relFileName)
-
-	// looking for all go files that have export func
-	// using in entrypoint
-	err = filepath.Walk(filepath.Dir(fileName), visit(&files))
```

Comment:
I'm wondering if we should keep this logic, not for file watching but for tracing the ast to find `readFile()` calls the same way we do in `@vercel/node`.

cc @TooTallNate since we recently found out `@vercel/go` doesnt include assets in the lambda

Reply:
I suggest we make a card to fix that behavior and restore this logic when it can be adapted for that purpose. The changes as written fixes a different issue.

Reply:
I made a draft card for this in our board at the top of the backlog.